jkube-kit/doc/src/main/asciidoc/inc/helm/_jkube_helm.adoc

```diff
@@ -101,10 +101,18 @@ endif::[]
 |
 
 | *icon*
-| The Chart URL to an SVG or PNG image to be used as an icon , default is extracted from the kubernetes manifest
+| The Chart URL to an SVG or PNG image to be used as an icon, default is extracted from the kubernetes manifest
 (`kubernetes.yml`) `jkube.eclipse.org/iconUrl` annotation if not provided.
 | `jkube.helm.icon`
 
+| *appVersion*
+| The version of the application that Chart contains
+(`kubernetes.yml`) `jkube.eclipse.org/iconUrl` annotation if not provided.
```

Comment:
```suggestion
version of the application
```

Reply:
:flushed: not sure how this ended up this way